torch/csrc/autograd/profiler.cpp

```diff
@@ -437,6 +462,49 @@ static jit::CodeTemplate event_template(R"(
   "args": {}
 })");
 
+void writeProfilerEventsToStream(std::ostream& out, const std::vector<Event*>& events) {
+  TORCH_CHECK(out, "Could not open file");
+  Event* profiler_start = nullptr;
+  for (Event* e : events) {
+    if (0 == strcmp(e->name(), "__start_profile")) {
+      profiler_start = e;
+      break;
+    }
+  }
+  TORCH_CHECK(profiler_start, "Could not find __start_profile mark");
+
+struct PairHash {
+  size_t operator()(std::pair<at::RecordFunctionHandle, int> p) const noexcept {
+    return std::hash<unsigned long>()(p.first) ^ std::hash<int>()(p.second);
```

Comment:
unsigned long -> RecordFunctionHandle, int -> int64_t

Reply:
Actually, just realizing, we don't need int64_t just for node_it, we can probably use uint16_t